Guides.md

```diff
@@ -43,7 +53,8 @@ On OS X you will need sshfs installed. The easiest way to install sshfs is to in
 After that you can do:
 
 	$ mkdir CYD-remote
-	$ sshfs <liu-id>@ssh.cyd.liu.se:/cyd/home/<liu-id> CYD-remote
+	$ sshfs <liu-id>@ssh.cyd.liu.se: CYD-remote
```

Comment:
Kommandot är kortare att skriva. Vi kan ändra tillbaka om det inte gillas men själv skulle jag skriva mitt kommando så :) 

> 5 juni 2016 kl. 08:28 skrev Rovanion Luckey <notifications@github.com>:
> 
> In Guides.md <https://github.com/CYD-poolen/cyd.liu.se/pull/14#discussion_r65812050>:
> 
> > @@ -43,7 +53,8 @@ On OS X you will need sshfs installed. The easiest way to install sshfs is to in
> >  After that you can do:
> >  
> >  	$ mkdir CYD-remote
> > -	$ sshfs <liu-id>@ssh.cyd.liu.se:/cyd/home/<liu-id> CYD-remote
> > +	$ sshfs <liu-id>@ssh.cyd.liu.se: CYD-remote
> Varför är det bättre att inte peka ut hemmappen?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/CYD-poolen/cyd.liu.se/pull/14/files/df9abb04173face4c75ec747a0acfe0c43d808c8#r65812050>, or mute the thread <https://github.com/notifications/unsubscribe/ADsiMEoP3qqBxdDqv8yfedTSvMTYK7QYks5qImyKgaJpZM4IuIcF>.
>